src/pocketmine/OfflinePlayer.php

```diff
@@ -44,7 +44,7 @@ class OfflinePlayer implements IPlayer, Metadatable{
 	public function __construct(Server $server, string $name){
 		$this->server = $server;
 		$this->name = $name;
-		if(file_exists($this->server->getDataPath() . "players/" . strtolower($this->getName()) . ".dat")){
+		if(file_exists($this->server->getDataPath() . "players/" . $this->name . ".dat")){
```

Comment:
It's possible that an `OfflinePlayer` instance may be constructed by something else (such as a unit test), in which case the name might not be passed in lowercase. The class should not rely on the caller to do its work for it. At the least this check should be `strtolower()`d.